README.md

```diff
@@ -13,8 +13,10 @@ It presents users with a familiar and consistent experience when accessing their
 <br />
 
 ## Table of Contents
-
-- [How to use the PowerDMS File Picker](#Using-the-file-picker)
+- [Integrating the PowerDMS File Picker](#Integrating-the-PowerDMS-File-Picker)
+  - [Authenticating via the File Picker](#Authenticating-via-the-File-Picker)
+  - [Configure an endpoint to renew tokens]()

```

Comment:
`[Configure an endpoint to renew tokens]()` seems to be missing a url?

Reply:
Good catch, fixed and also fixed the order to reflect the actual section order